### DISCUSSION TITLE
Can I put more than one group to receive the apk?

### DISCUSSION THREAD

**Discussion by thiagomatheusms** [wzieba/Firebase-Distribution-Github-Action#42] [Q&A] [answered]
I have tried put two groups to receive the apk but gets a warning that a string is expected and not an array.

follow an example: 

groups: 
                - GROUP1
                - GROUP2

**Comment by wzieba** [comment, marked as answer]
Yes you can - but please do this by single argument with coma-separated groups.

```
groups: "GROUP1, GROUP2"
```
this should work fine.

For more details you can check Firebase CLI docs, we just pass arguments from action to the CLI: https://firebase.google.com/docs/app-distribution/android/distribute-cli

**Reply by thiagomatheusms** [reply]
Thanks!
